src/http-gateways/path-gateway.md

```diff
@@ -542,6 +542,46 @@ Optional, present in certain response types:
   non-executable binary response types are not used in `<script>` and `<style>`
   HTML tags.
 
+### `Ipfs-Gateway-Features` (response header)
+
+Optional, this header SHOULD be only returned in response to HTTP `OPTIONS` request.
```

Comment:
(just tuning into this PR now that I care about signalling, sorry!)


Whatever this header is, I think it should also be returned when a server doesn't implement a feature requested, either via `Accept` header or other means and in that case the server should return a `415`. That way I can parse the list of `Accept` content types, and be strict about the parameters, and if I don't find one that I fully support, return a `415` with this signal that says what I do support.

That way, a client can optimistically make a request, but be prepared to downgrade if it doesn't get what it wants. Rather than needing to `OPTIONS` first, thereby saving a round-trip.